File: Sportski_centar_androidStudio/app/src/main/java/com/example/sports_centar/data/GetProstorijeVM.java
package com.example.sports_centar.data;

import java.io.Serializable;
import java.util.List;

public class GetProstorijeVM implements Serializable {
    public List<Row> rows;

    public class Row implements Serializable
    {
        public int ProstorijaID;
        public String Naziv;
    }
}

File: SportskiCentarRKS_API/SportskiCentarRKS_API/ViewModels/GetProstorijeVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SportskiCentarRKS_API.ViewModels
{
    public class GetProstorijeVM
    {
        public List<Row> rows { get; set; }

        public class Row
        {
            public int ProstorijaID { get; set; }
            public string Naziv { get; set; }
        }
    }
}